locations_test.go

```diff
@@ -97,6 +97,26 @@ func TestLocationsService_Update(t *testing.T) {
 	require.Equal(t, fakeLocation, actual)
 }
 
+func TestLocationsService_Patch(t *testing.T) {
+	data := LocationPatchRequest{
+		IsDefault: false,
```

Comment:
`false` - is default value for boolean in GO, so maybe better `true` to ensure that PATCH sends correct values ?

Reply:
OK, I've updated test to make it more useful.